cafebabel/static/js/edit-article.js

```diff
@@ -1,36 +1,48 @@
-const tagButtonAdd = document.querySelector('.tags button.add')
-const tagButtonRemove = document.querySelector('.tags button.remove')
-const tagsList = document.querySelector('.tags .tagsList')
+class Tags {
+  constructor(domContext) {
```

Comment:
the convention that has been taken in this project is to have a space after the name when declaring a function name. Let's keep this uniform.

Reply:
@vinyll - I think I'm missing something here.
Do you ask to add a space between function name and args ?
Like this:
 constructor (domContext) - instead of: constructor(domContext)

Reply:
It is now handled by prettierjs